pages/index.js

```diff
@@ -11,10 +12,15 @@ const Modal = dynamic(() => import('../src/components/Form/Modal'), {
 	ssr: false,
 });
 
-const Map = dynamic(() => import('../src/components/Map/Map'), { ssr: false });
+const ReviewsMap = dynamic(() => import('../src/components/Map/ReviewsMap'), { ssr: false });
 
 export default function Home() {
-	const editmode = useStore(state => state.editmode);
+	//This hideEdit function sets the edit mode to false. In previous versions of my app, the edit mode was set to false only after submitting the form, but technically, a user could click on edit and then decide that they do not want to edit and go back to the homepage or to the profile page instead. In such a case, the editmode remains true. This affected the rendering of my map in the form page (since the map shows the default coordinates (around Berlin) when entering the form page to add a new review, but if one is editing an existing review, the map displays the approximate location of the review to be edited instead).
+	const hideEdit = useStore(state => state.hideEdit);
+	useEffect(() => {
+		hideEdit();
+	}, [hideEdit]);
+
```

Comment:
```suggestion
	useEffect(() => {
		useStore.getState().hideEdit();
	}, []);

```

This is something the zustand API allows you to do, so you dont need to fill in the dependency array.